### PR TITLE
Basic support for two-team matches

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This works for two-team matches, though there are still some things here which aren't fully generalised to arbitrary numbers of teams in an arena.

Fixes https://github.com/srobo/competition-team-minutes/issues/255

### CODE REVIEW

File: comp/schedule.html [hunk 1]
```diff
@@ -69,10 +69,9 @@ <h2>[[ session.description ]]</h2>
         <tr>
             <th data-ng-if="!session.show_arena_title">Time</th>
             <th data-ng-if="!session.show_arena_title">Match</th>
-            <th data-ng-repeat-start="arena in session.arenas">Corner 0</th>
-            <th>Corner 1</th>
-            <th>Corner 2</th>
-            <th data-ng-repeat-end>Corner 3</th>
+            <th data-ng-repeat="corner in session.all_corners track by $index">
```
Comment (sedders123) [inline comp/schedule.html:72]: Just want to double check my understanding on this. We're using `all_corners` as we want to handle having multiple arenas. We don't need to update the body of this table as it already renders a column for each team in the match, and we only need to repeat over the teams in the match as a match can have multiple arenas so `match.teams` would contain the teams across all arenas?

**Review by sedders123 — APPROVED** [review]
This looks good to me though I haven't run this locally as I don't have a compstate or the HTTP interface setup. I have followed the code through and it all makes sense (one part I wanted to double check my understanding of though).

File: comp/schedule.html (hunk 1 above)
Comment (PeterJCLaw) [reply comp/schedule.html:72]: Yes.

Comment (sedders123) [reply comp/schedule.html:72]: Thanks :)

**Pull request merged**